_sass/components/_buttons.scss

```diff
@@ -22,4 +23,104 @@
     //box-shadow: 0 7px 14px rgba(50, 50, 93, 0.1), 0 3px 6px rgba(0, 0, 0, 0.08);
```

Comment:
```suggestion
```

Reply:
I didn't want to change the legacy code that was copied from Serif (and it's gonna be deprecated anyways).